pybliometrics/scopus/serial_title.py

```diff
@@ -264,3 +274,54 @@ def _parse_list(d, metric):
         return sorted(set(values))
     except (KeyError, TypeError):
         return None
+
+
+def _get_all_cite_score_years(self, named_info_list, named_rank_list, data) -> Optional[List[NamedTuple]]:
+    """Auxiliary function to get all information contained in cite score information list for the `CITESCORE` view"""
```

Comment:
Split into two lines, add full stop and then the last quotation marks on a newline